### PR TITLE
Book/features

### PR DESCRIPTION
**Pull request by techwolf404**
### Catalog: Book, Label Classes and methods
- Created a Book class in a separate .rb file.
- Created a Label class with an association to the Item class (in a separate .rb file).
- All Book class properties visible in the diagram have been defined and set up in the constructor method.
- All Label class properties visible in the diagram have been defined and set up in the constructor method.
##### Implement methods:
- add_item method in the Label class
  1.   hould take an instance of the Item class as an input
  2. should add the input item to the collection of items
  3. should add self as a property of the item object (by using the correct setter from the item object)
- can_be_archived?() in the Book class
  1. should override the method from the parent class
  2. should return true if parent's method returns true OR if cover_state equals to "bad"
  3. otherwise, it should return false

### CODE REVIEW

File: book.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+       puts 'Enter the book publisher:'
+       publisher = gets.chomp
+       puts 'Enter the book cover state:'
+       cover_state = gets.chomp
+         book = Book.new(publisher, cover_state)
+         return book
+         puts '--------------------------------
+         Book created successfully
+         --------------------------------'
+    end
```
Comment (Nizamuddin4493) [inline book.rb]: Please I will suggest you move this code to the App.rb, No need for it here.

**Comment by Nizamuddin4493** [comment]
### Hi @techwolf404,
Nice job implementing the requested changes but we have to fix something minor here;

Status: Changes Requested ⛔
196655262-19b91a00-2ff4-4aba-b215-637cd1cd9e97

### Highlights
Good job with your Book class 👍

### Required Changes
- Please I will suggest you to move this code to the App.rb, No need it here.

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding! clapclapclap

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**Review by Nizamuddin4493 — CHANGES_REQUESTED** [review]

File: book.rb (hunk 1 above)
Comment (techwolf404) [reply book.rb]: I have implemented the changes. Added the app too

**Comment by Nizamuddin4493** [comment]
# STATUS: APPROVED 🟢 👏
### Hi @techwolf404  🙋‍♂️,
#### Your project is complete! There is nothing else to say other than... Great job 👏
### Congratulations! 🎉

minions-90df4f92fe48ea1ff0c334326b5709ef

## TO HIGHLIGHT:

- Great job implementing the changes by the previous reviewer ✅
- Your project is professional ✅
- Your repo is professional ✅
- Well structured files ✅
- Professional README.md file 💯

Now you can merge this branch and move on 🥇
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES 🤞 🍀

**Pull request merged**